salt/states/junos.py

```diff
@@ -272,15 +302,15 @@ def shutdown(name, **kwargs):
 
 
 @resultdecorator
-def install_config(name, **kwargs):
+def install_config(name, path, **kwargs):
```

Comment:
Why adding path when name provided the 
286                 - name: salt://configs/interface.set

do not see reason to change label of field, yes it is a path but 'name' is used elsewhere in Salt Doc. too for a provided path.

Reply:
reverted via https://github.com/saltstack/salt/pull/57281/commits/682cb0a08baa14e1fa4684163c5588cbda672d25#diff-442286d51776ddefc791bc7f15e42fc2R285